controllers/dishes/getDishes/index.js

```diff
@@ -0,0 +1,7 @@
+import { Dish } from '../../../models/index.js';
+
+export const getDishes = async (_req, res) => {
+  const dishes = await Dish.find({});
```

Comment:
without any parameters for filter, why?

Reply:
It might sound silly , but I didn't add query parameters initially because our document with routes didn't include them. But now, I've did it. Really thank you)